theme/templates/resource-landing-page/title-section.html

```diff
@@ -51,6 +51,12 @@
                         <br>
                         <span>Content files are required to publish the resource or make the resource public or discoverable.</span>
                 {% endif %}
+                {%  if cm.resource_type|lower == "collectionresource" %}
+                    {% if not cm.has_resources%}
+                        <br>
+                        <span>Collection should contain at least one resource to be made it published, public or discoverable.</span>
+                    {% endif %}
+                {% endif %}
```

Comment:
@pkdash I can duplicate the issue @hyi is referring here: after creating a new collection, once user adds abstract and keywords the "Metadata Status: sufficient to publish or make public" popup shows up. But the collection has no contained resources yet.